deploy/melange.yaml.tmpl

```diff
@@ -59,6 +72,5 @@ pipeline:
       mv bin/kotsadm "${DESTDIR}/kotsadm"
       mv bin/kots "${DESTDIR}/kots"
 
-      ln -s /usr/bin/helm ${DESTDIR}/usr/local/bin/helm
```

Comment:
you'll also need to remove the helm binary from the apko file: https://github.com/replicatedhq/kots/blob/main/deploy/apko.yaml.tmpl#L16